### PR TITLE
Fix InterRDF_s normalization problem when given density = True

### LINKED ISSUE #120: Wrong results from InterRDF_s when given 'density=True'
### Expected behaviour
Give correct single-atom to single-atom RDF when given 'density=True'.

### Actual behaviour
The rdf will be 1/N of the correct value. 
Where N = nA * nB, and nA, nB are the number of atoms in atomgroup A and atomgroup B.

### Code to reproduce the behaviour

``` python
import MDAnalysis as mda
from MDAnalysisTests.datafiles import GRO_MEMPROT, XTC_MEMPROT
u = mda.Universe(GRO_MEMPROT, XTC_MEMPROT)

s1 = u.select_atoms('name ZND and resid 289')
s2 = u.select_atoms('(name OD1 or name OD2) and resid 51 and sphzone 5.0
                         (resid 289)')
s3 = u.select_atoms('name ZND and (resid 291 or resid 292)')
s4 = u.select_atoms('(name OD1 or name OD2) and sphzone 5.0 (resid 291)')
ags = [[s1, s2], [s3, s4]]

rdf = InterRDF_s(u, ags, density=True)
rdf.run()



....

```

### Currently version of PMDA:
 0.2.1

### PR DESCRIPTION
**Pull request by VOD555**
Fixes #120 

Changes made in this Pull Request:
 - Change the N=nA * nB to be 1, as this is a single-atom to single-atom RDF calculaton.

PR Checklist
------------
 - [x] Tests?
 - [x] Docs?
 - [x] CHANGELOG updated?
 - [x] Issue raised/referenced?

### CODE REVIEW

**Comment by VOD555** [comment]
Tests for density function failed as described in #122 .

File: pmda/test/test_rdf_s.py [hunk 1]
```diff
@@ -41,6 +41,14 @@ def rdf_s(u, sels, scheduler):
     return InterRDF_s(u, sels).run()
 
 
+@pytest.fixture(scope='module')
+def rdf_ref(u):
```
Comment (orbeckst) [inline pmda/test/test_rdf_s.py:45]: Is the reference the normal RDF calculation with a single atom in each group?

File: pmda/rdf.py [hunk 2]
```diff
@@ -318,9 +318,9 @@ def _conclude(self):
         # Empty lists to restore indices, RDF
         rdf = []
 
-        for i, (nA, nB) in enumerate(self.ag_shape):
+        for i in range(len(self.ag_shape)):
```
Comment (orbeckst) [inline pmda/rdf.py:321]: Is this also broken in `MDAnalysis.analysis.rdf.InterRDF_s`?

**Comment by orbeckst** [comment]
I merged master.

**Comment by orbeckst** [comment]
Update the CHANGELOG and fix the PEP8 issues and then this should be good to go. Or do you have any other concerns, @VOD555?

**Pull request merged**